casepro/contacts/models.py

```diff
@@ -430,7 +440,7 @@ def as_json(self, full=True):
         """
         Prepares a contact for JSON serialization
         """
-        result = {'id': self.pk, 'name': self.get_display_name()}
+        result = {'id': self.pk, 'name': self.get_display_name(), 'urns': self.get_display_urns()}
```

Comment:
Ah I see. I've been using the case object to get all of the contact info. Thanks